iquip/apps/scheduler.py

```diff
@@ -2,14 +2,17 @@
 
 import enum
 import functools
+import json
 import logging
 from typing import Any, Optional, Sequence, Tuple
 
 import requests
+import websockets
 from PyQt5.QtCore import (
     pyqtSignal, pyqtSlot, QAbstractTableModel, QModelIndex, QObject, Qt, QThread, QVariant
 )
 from PyQt5.QtWidgets import QAction, QPushButton, QTableView, QVBoxLayout, QWidget
+from websockets.sync.client import connect
```

Comment:
I think since you already imported `websockets`, you don't have to import `connect` again (also it is not used many times).
Moreover, this module (`scheduler`) is not focused that much in `websockets` so `connect()` itself might seem confusing (although in this code you used proper context manager name so one can infer that it is for websockets easily).

Reply:
I see! Then, I will import only `websockets` module.

Reply:
Umm.. I don't know the exact reason, but if importing only this module, `connect()` cannot be found.

![image](https://github.com/snu-quiqcl/iquip/assets/76851886/9674cc51-f414-4d65-bc41-3c9224f0e5c3)

Temporarily, I left it as it was.

Reply:
Oh, I see. Then it should be imported with full path.
In that case, let's keep the current code. It is readable enough!